.pullquester/pullrequest.json

```diff
@@ -19,10 +19,12 @@
         {
             "name": "Joel Brighton",
             "value": "@joebri"
-        },
+        }
+    ],
+    "testers": [
         {
-            "name": "Jonathan Park",
-            "value": "@park9140"
+            "name": "Phillip Balsley",
```

Comment:
FYI @hyperd @pbalsley  does not sign off on style guide changes. So no need to add him here.